tests/test_modules/thread_aware_modules.py

```diff
@@ -102,3 +102,37 @@ def PreProcess(self) -> None:
 
   def PostProcess(self) -> None:
     self.logger.info("ThreadAwareConsumerModule Static Post Process")
+
+class Issue503Module(module.ThreadAwareModule):
+  """This is a module for testing a certain pattern of container handling.
+
+  As described by https://github.com/log2timeline/dftimewolf/issues/503 this
+  module pops containers for input, and uses the same container type as output.
+  """
+  def __init__(self, state, name=None):
+    super(Issue503Module, self).__init__(state, name)
+
+  def SetUp(self): # pylint: disable=arguments-differ
+    """SetUp"""
+    self.logger.info('{0:s} SetUp!'.format(self.name))
```

Comment:
Prefer using f-strings over format strings, they're faster and their syntax is easier to read. Same for other format strings in this file.

Reply:
As discussed in PM since this is all over this file, lets change this in another PR.

Reply:
Raised issue #505 for wider discussion on this.